### DISCUSSION TITLE
Join the GitHub Copilot waitlist

### DISCUSSION THREAD

**Discussion by senyu-up** [community/community#15638] [Copilot Conversations] [answered]
<img width="622" alt="image" src="https://user-images.githubusercontent.com/42713450/165351911-fe99266b-c21b-48f3-8acd-d4bd765e2835.png">
I really want to use this plugin and I have been applying for a week, how can the author notice me and give me the qualification to pass the internal test.

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
